### PR TITLE
Changed quit and close file menu item's names and colocated them to bottom

### PR DESCRIPTION
**Pull request by nalin243**
Resolves: #6644 

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

**Comment by kryksyh** [comment]
@nalin243 please rebase on the latest master, to get the ci build working.

**Comment by nalin243** [comment]
@kryksyh 

Should I let this be or close and open a new request?

**Comment by kryksyh** [comment]
> @kryksyh
> 
> Should I let this be or close and open a new request?

you can keep this pr 👍

**Comment by petersampsonaudacity** [comment]
Testing on W10 with @nalin243 's latest branch build for this:  audacity-win-3.7.0-alpha-20240908+575662c-x64

image

a)The new wording is implemented properly
b) the two commands are now co-located at the bottom of the File menu

But I'm failing to understand why we have (or need) a separator line between the two commands as they are both very closely related in functionality.

File: src/menus/FileMenus.cpp [hunk 1]
```diff
@@ -553,12 +549,16 @@ auto FileMenu()
          )
       ),
 
+      Section( "Close", Command( wxT("Close"), XXO("&Close Project"), OnClose,
```
Comment (LWinterberg) [inline src/menus/FileMenus.cpp:552]: you don't need the `Section("Close"` bit anymore, you can instead put the `Command(...)` thing into the Exit section.

File: src/menus/FileMenus.cpp [hunk 2]
```diff
@@ -553,12 +549,16 @@ auto FileMenu()
...
+            AudioIONotBusyFlag(), wxT("Ctrl+W") )
+      ),
+
       Section( "Exit",
          // On the Mac, the Exit item doesn't actually go here...wxMac will
          // pull it out
          // and put it in the Audacity menu for us based on its ID.
          /* i18n-hint: (verb) It's item on a menu. */
```
Comment (LWinterberg) [inline src/menus/FileMenus.cpp:560]: That said, watch the comments. If the entire section is relocated to the App menu on macOS, you'll need to do some platform specific code via

```cpp

#ifdef __WXMAC__
  macos code
#else 
  windows/linux code
#endif
```

**Comment by DavidBailes** [comment]
> But I'm failing to understand why we have (or need) a separator line between the two commands as they are both very closely related in functionality.

@petersampsonaudacity 
Standard practice in editors appears to have such commands in separate sections. (Can you find one that doesn't?) Just because two commands both close something, doesn't mean they should be in the same section. Closing a project (perhaps one of many) is quite different from closing the program completely.

File: src/menus/FileMenus.cpp (hunk 2 above)
Comment (nalin243) [reply src/menus/FileMenus.cpp:560]: I am not really familiar with mac but what I am understanding is that if I put both 'Quit Audacity' and 'Close Project' in the same section then to make sure that mac doesn't put 'Close Project' in the app menu as well, they would need to be in different sections. Won't that result in the section dividing line still being present in mac then?

**Comment by nalin243** [comment]
@DavidBailes @LWinterberg 

Should I keep it in the same section or let it be then?

**Comment by DavidBailes** [comment]
> @DavidBailes @LWinterberg
> 
> Should I keep it in the same section or let it be then?

It's up to @LWinterberg , not me.

**Review by LWinterberg — APPROVED** [review]
let's leave it like this - the additional separator also makes it slightly more difficult to misclick quit instead of close.

**Comment by petersampsonaudacity** [comment]
I assume this will be merged for 3.7. 0 master and not the impending 3.6.4 ?

**Comment by kryksyh** [comment]
@nalin243 could you rebase your PR on the latest master please?

**Pull request closed without merging**